src/utils/schema.js

```diff
@@ -13,3 +13,29 @@ exports.getModelName = (model) => model.modelName;
 
 // TODO: Remove nameOld attribute once the lianas versions older than 2.0.0 are minority
 exports.getModelNameOld = (model) => model.collection.name.replace(' ', '');
+
+const getNestedFieldType = (mongooseSchema, nestedFieldPath) => {
```

Comment:
Based on a field path, this function introspect the mongoose schema and attempt to reach the field filtered on. If the field is reachable, the function returns its type